### DISCUSSION TITLE
How do I enable a Checked context from .csproj file?

### DISCUSSION THREAD

**Discussion by CharlesTaylor7** [dotnet/csharplang#4226] [Q&A] [answered]
Is there a way to set a default checked context for my whole assembly from the `.csproj` file?
More generally, what's the best way to customize compiler options if I'm using VSCode and not Visual Studio?

The documentation on this page only goes over what to do in Visual Studio:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/checked-compiler-option

**Comment by Joe4evr** [comment, marked as answer]
`<CheckForOverflowUnderflow>True</CheckForOverflowUnderflow>`
